portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -2,6 +2,40 @@ definition {
 
 	var rowNumber = "1";
 
+	macro addParagraphLinkToAURL {
+		Click(
+			key_titleName = "Link",
+			locator1 = "FormFields#TOOLBAR_ANY_ITEM_BUTTON");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "FormFields#PARAGRAPH_LINK_MODAL");
+
+		SelectFrame(value1 = "relative=top");
+
+		Type(
+			fieldLabel = "URL",
+			locator1 = "FormFields#PARAGRAPH_LINK_TEXT_INPUT",
+			value1 = "${paragraphLink}");
+
+		if (isSet(displayText)) {
+			Type(
+				fieldLabel = "Display Text",
+				locator1 = "FormFields#PARAGRAPH_LINK_TEXT_INPUT",
+				value1 = "${displayText}");
+		}
+
+		if (isSet(linkProtocol)) {
```

Comment:
@leticiachiu, what is the purpose of `linkProtocol`? Is it necessary to fix the test?